src/NServiceBus.AmazonSQS/Configure/SqsTransportSettings.cs

```diff
@@ -170,6 +170,19 @@ public static TransportExtensions<SqsTransport> NativeDeferral(this TransportExt
             return transportExtensions;
         }
 
+        /// <summary>
+        /// Configures the SQS transport to be compatible with 1.x versions of the transport.
+        /// </summary>
+        /// <param name="transportExtensions"></param>
+        /// <param name="use">Set to true to use allow the endpoint to send messages to endpoints using version 1.x of the transport; false otherwise.</param>
+        /// <returns></returns>
+        public static TransportExtensions<SqsTransport> V1CompatibilityMode(this TransportExtensions<SqsTransport> transportExtensions, bool use = true)
```

Comment:
If by default the endpoint is not backward compatible what’s the point of setting it to `false`?
Shouldn’t this extension method simply be something like `EnableV1CompatibilityMode()` with no arguments? (Or `RunInV1CompatibilityMode()`)

Reply:
@mauroservienti the intent _is_ to be backwards compatible by default and opt out when that compatibility is no longer needed.

Reply:
As far as I can tell this is not what the code does, by default `V1CompatibilityMode` is not set thus it’ll be `false`.
And in https://github.com/Particular/NServiceBus.AmazonSQS/pull/214#issuecomment-386191689 it seems to me we stated that by default v3 is NOT backward compatible, which means that compatibility with v1 needs to be explicitly turned on.

Am I missing something obvious?

Reply:
The other methods had the same default variable value, so I went with it. I am happy to remove it.